indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -163,7 +166,7 @@ def _check_revision_access(self):
 
     def _process(self):
         args = parser.parse({
-            'files': EditingFilesField(self.event, allow_claimed_files=True, required=True)
+            'files': EditingFilesField(self.event, contrib=self.contrib, allow_claimed_files=True, required=True)
```

Comment:
I'd use positional syntax instead of the kwarg, since `contrib` is required anyway and somewhat clear even without the param name here (and in the other places where we're using this field)